acquire/acquire.py

```diff
@@ -683,7 +681,7 @@ def _run(cls, target: Target, cli_args: argparse.Namespace, collector: Collector
 
         patterns = ["$Recycle.bin/*/$I*", "Recycler/*/INFO2", "Recycled/INFO2"]
 
-        if not cli_args.no_data_files:
+        if cli_args.data_files is None and cli_args.data_files:
```

Comment:
If it's `None` then `cli_args.data_files` evaluates to `False`, so I guess you mean `or` here?